backend/routes/flightStatus.js

```diff
@@ -1,46 +1,247 @@
-import { API_KEY, API_SECRET } from "../config.js";
+import { API_KEY, API_SECRET, GOOGLE_MAPS_API_KEY } from "../config.js";
 import Amadeus from "amadeus";
 import express from "express";
-import { PrismaClient } from '@prisma/client'; 
+import { PrismaClient } from '@prisma/client';
 import { authenticateUser } from '../auth.js';
+import axios from 'axios';
 
 const prisma = new PrismaClient();
 const router = express.Router();
 const amadeus = new Amadeus({
-    clientId: API_KEY,
```

Comment:
Do we have autoformatting enabled in VSCode? It would be good if we can get rid of those nit changes for future pull requests, so that we can focus review on important changes only.